app/models/post.rb

```diff
@@ -51,13 +51,15 @@ class Post < ApplicationRecord
   update_algolia 'AlgoliaPostsIndex'
   embed_links_in :content, to: :embed
 
+  enum locked_reason: { SPAM: 0, TOO_HEATED: 1 }
```

Comment:
These should be `{ spam: 0, too_heated: 1 }` and then mapped from lowercase symbol in GraphQL

Rails and GraphQL case their enums differently 😅 Rails also generates methods, so you could do `post.SPAM?` with this, which just looks silly

Reply:
`post.SPAMSPAMSPASMAPSAPSMAPMS`